### DISCUSSION TITLE
Subscription Closed Exception

### DISCUSSION THREAD

**Discussion by bmills74** [ravendb/ravendb#16695] [Q&A]
We're getting the following exception happening on a daily basis with a couple different subscriptions

"Message": "Subscription With Id 'UserDocumentSubscription' was closed.  Raven.Client.Exceptions.Documents.Subscriptions.SubscriptionClosedException: Dropped by API request (request ip:51.140.70.151, 

Presumably it should queue  any unprocessed documents for when it comes back up, but we are seeing cases of documents not being processed, so this exception is the number one suspect for this problem.
Any knowledge on the impact of this exception and why it's happening would be most appreciated.

**Comment by ayende** [comment]
Someone (see the ip there) closed the subscription (probably from the studio).
That means that it will not try to reconnect
